Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -750,6 +854,29 @@ void WiiSockMan::Convert(sockaddr_in const& from, WiiSockAddrIn& to, s32 addrlen
     to.len = addrlen;
 }
 
+void WiiSockMan::DoState(PointerWrap& p)
+{
+  bool saving =
+      p.mode == PointerWrap::Mode::MODE_WRITE || p.mode == PointerWrap::Mode::MODE_MEASURE;
+  auto& target = saving ? pending_polls : failed_polls;
+  auto size = target.size();
+  p.Do(size);
+  if (!saving)
+    target.resize(size);
+  std::for_each(target.begin(), target.end(), [&](auto& pcmd) {
```

Comment:
- Is it useful to save the `timeout` and `trigger_count` value when they will error out anyway?
 - Can't on a savestate load `POLLHUP | POLLERR` be assigned to all `pending_polls.wii_fds.revents` rather than creating a new vector?